.checkstyle/checkstyle.xml

```diff
@@ -107,7 +107,7 @@
         </module>
         <module name="CyclomaticComplexity">
             <!-- default is 10-->
-            <property name="max" value="19"/>
+            <property name="max" value="20"/>
```

Comment:
Would annotation on the class/method infringing this be better than increasing the limit for every class? We normally use annotations in other projects.

Reply:
Ok, I reverted and moved warning suppression into the code.